tests/sqlparser_postgres.rs

```diff
@@ -893,6 +893,20 @@ fn parse_drop_schema_if_exists() {
     }
 }
 
+#[test]
+fn parse_drop_database_if_exists() {
```

Comment:
can this test case also live in common.rs similar to the other drop test case?

Reply:
Done!